server/controllers/getReferrals/index.ts

```diff
@@ -42,21 +42,74 @@ const getReferrals =
 
     if (!rows) {
       logger.error(
-        'Referrals for %s cannot be fetched in the referrals table',
+        'Level one referrals for %s cannot be fetched in the referrals table',
         address,
       );
       throw new TRPCError({
         code: 'UNPROCESSABLE_CONTENT',
-        message: 'Referrals cannot be fetched',
+        message: 'Level one referrals cannot be fetched',
       });
     }
 
+    const levelOneReferrals = rows.map(({ referral }) => referral);
+
+    const placeholders = new Array(levelOneReferrals.length)
+      .fill('?')
+      .join(',');
+    try {
+      rows = db
+        .query<
+          dbResponse,
+          string[]
+        >(`SELECT referral FROM referrals WHERE referrer IN (${placeholders})`)
+        .all(...levelOneReferrals);
```

Comment:
this may be broken 
you can use single query to get two levels

```
SELECT 
	r.referrer,
	r.referral,
	r2.referral
FROM referrals r
LEFT JOIN referrals r2 ON (r2.referrer = r.referral)
WHERE r.referrer = '?'
``` 
the response would be like 
<img width="411" alt="image" src="https://github.com/hadronlabs-org/drop-points-crawler/assets/198887/ba0cc5aa-f900-4272-a57b-8b22512d3109">